game/game.go

```diff
@@ -167,6 +167,9 @@ func (g *Game) handleCommand(c playerCommand) {
 }
 
 func (g *Game) resign(p *player) {
+	if p.kibitzer {
+		return
```

Comment:
Perhaps :) instead of just ignoring the command, this could be turned into a chat command "Kibitzer {name} suggests someone resign"

Reply:
I love it!  Added it in ea6388bf54529e62f1b66f6399899394897a2d28.  Testing this feature even uncovered a nasty (game crashing) bug I missed and fixed in b1cefcaac9b93fecb820f70853e986daf8f16691.

I knew it was too good to be true that everything worked on the first try.  As is usually the case, the last 20% ended up being the hardest, but I'm glad I fixed that bug before pushing this to the server!